<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookNow</class>
 <widget class="QDialog" name="BookNow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#BookNow{
background-image: url(:/images/images/BookNow_bgNew.png) ;
}
*{
	color:black;
}

.QLineEdit
{
border-radius: 10px ;


}</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat:no-repeat;
background-image: url(:/images/travelSphere.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>551</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background-color: rgb(170, 170, 0);
border-radius: 15px; 
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="DepartueDate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Departure Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReturnDate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Return Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="departDate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background:white;</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>10</month>
      <day>26</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd/M/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="returnDate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>12</month>
      <day>23</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="destination">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#destination{
color: black ; 
background-color:white;
border-radius:10px;

}</string>
   </property>
   <item>
    <property name="text">
     <string>Chandragiri</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thousand Steps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pokhara</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nagarkot</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>651</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ContactUs_PushButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
backgound: yellow ; 
border-radius: 5px ;
</string>
        </property>
        <property name="text">
         <string>Contact Us</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
backgound: yellow ; 
border-radius: 5px ;
</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Profile_PushButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:9px;
	background-color: qlineargradient(spread:pad, x1:0.0788177, y1:0.380682, x2:1, y2:0, stop:0.0640394 rgba(250, 130, 78, 255), stop:1 rgba(224, 206, 107, 255));
color:white;
	font: 13pt &quot;Monaco&quot;;
	border-color: rgb(0,0,0);
  border-style: solid;
  border-width: 1px;

}

*:hover{
color:#F5E0E0;
}</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="From">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="book_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#book_button{
border-radius: 10px ;
background-color:rgb(85, 170, 255);

}</string>
   </property>
   <property name="text">
    <string>Confirm Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="From_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Company username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>671</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="see_Companies">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
backgound: yellow ; 
/*background-image: url(:/images/images/ icon _home_.png) ;*/
border-radius: 5px ;
</string>
   </property>
   <property name="text">
    <string>See Companies</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
